src/main/java/com/example/TicketBookingApp/controller/ControllerExceptionHandler.java

```diff
@@ -0,0 +1,17 @@
+package com.example.TicketBookingApp.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class ControllerExceptionHandler {
+
+    @ExceptionHandler
+    public ResponseEntity<String> handleValidationException(Exception exception){
+        return ResponseEntity
+                .status(HttpStatus.BAD_REQUEST)
```

Comment:
Czemu akurat BadRequest? Co oznacza ten kod błędu?

Reply:
Użyłem tego w celu ujednolicenia. Każdy z wyjątków przeze mnie zdefiniowanych występuje gdy klient prześle niepoprawne dane w zapytaniu.

Reply:
Tak, ale... Tutaj obsługujesz wszystkie `Exception`y ;) A nie każdy wyjątek to bad request.

Reply:
Fakt, dlatego lepiej gdybym zrobił, aby wpadały tu tylko wyjątki zdefiniowane przeze mnie. Zamiast _Exception exception_ powinna być lista argumentów np .: _value = SeatNotAvailableException.class_, (...)